indico/modules/events/registration/client/js/form/fields/AccompanyingPersonsInput.jsx

```diff
@@ -183,7 +185,7 @@ function AccompanyingPersonsComponent({
           </Label>
         )}
         {placesLimit !== Infinity && (
-          <div styleName="places-left">
+          <div id={id ? `${id}-placeslimit` : ''} styleName="places-left">
```

Comment:
`id=""` feels wrong. maybe `undefined` as a fallback is better? (but again, if we always have an ID you don't need the ternary there at all)